Libplanet.Tests/Store/Trie/KeyValueStoreTest.cs

```diff
@@ -77,6 +78,15 @@ public void Exists()
             Assert.False(KeyValueStore.Exists(randomKey));
         }
 
+        [SkippableFact]
+        public void ListKeys()
+        {
+            ImmutableHashSet<byte[]> keys = KeyValueStore.ListKeys().ToImmutableHashSet();
+            Assert.Equal(PreStoredDataCount, keys.Count);
+            var equalityComparer = new BytesEqualityComparer();
+            Assert.True(PreStoredDataKeys.ToImmutableHashSet(equalityComparer).SetEquals(keys));
```

Comment:
Can we use `Assert.Equal<T>(T expected, T actual, IEqualityComparer<T> comparer)`?

Reply:
If you intended code like below:

```suggestion
            Assert.Equals(PreStoredDataKeys, keys, equalityComparer);
```

It seems not able to cover not sorted elements like `SetEquals`.